modules/synthetic-liquidity-pools/src/lib.rs

```diff
@@ -37,9 +37,9 @@ pub trait Trait: system::Trait {
 decl_storage! {
 	trait Store for Module<T: Trait> as SyntheticLiquidityPools {
 		pub NextPoolId get(fn next_pool_id): LiquidityPoolId;
-		pub Owners get(fn owners): map hasher(blake2_128_concat) LiquidityPoolId => Option<(T::AccountId, LiquidityPoolId)>;
-		pub LiquidityPoolOptions get(fn liquidity_pool_options): double_map hasher(blake2_128_concat) LiquidityPoolId, hasher(blake2_128_concat) CurrencyId => Option<SyntheticLiquidityPoolOption>;
-		pub Balances get(fn balances): map hasher(blake2_128_concat) LiquidityPoolId => Balance;
+		pub Owners get(fn owners): map hasher(twox_64_concat) LiquidityPoolId => Option<(T::AccountId, LiquidityPoolId)>;
+		pub LiquidityPoolOptions get(fn liquidity_pool_options): double_map hasher(twox_64_concat) LiquidityPoolId, hasher(blake2_128_concat) CurrencyId => Option<SyntheticLiquidityPoolOption>;
```

Comment:
```suggestion
		pub LiquidityPoolOptions get(fn liquidity_pool_options): double_map hasher(twox_64_concat) LiquidityPoolId, hasher(twox_64_concat) CurrencyId => Option<SyntheticLiquidityPoolOption>;
```

Reply:
@xlc but CurrencyId is added by the user. would it be safe?

Reply:
yes. CurrencyId is an enum with very limited options. Any invalid CurrencyId will be rejected as invalid transaction during decode phase.

Also it is one byte, so max 256 possibilities, which is safe to hash collision resistance

Reply:
ok